x-pack/plugins/enterprise_search/public/applications/shared/types.ts

```diff
@@ -33,7 +33,7 @@ export interface SchemaConflicts {
 export interface IIndexingStatus {
   percentageComplete: number;
   numDocumentsWithErrors: number;
-  activeReindexJobId: number;
+  activeReindexJobId: string;
```

Comment:
Nice catch, thanks!

Approving on behalf of the App Search changes, I didn't check the WS changes as a heads up